tests/test_e2e.py

```diff
@@ -71,6 +71,41 @@ def test_tensorrtllm_e2e(self, trtllm_server, protocol):
         TritonCommands._infer(model, prompt=PROMPT, protocol=protocol)
         TritonCommands._profile(model, backend="tensorrtllm")
 
+    @pytest.mark.skipif(
+        os.environ.get("IMAGE_KIND") != "LLMAPI",
+        reason="Only run for TRT-LLM image with LLM-API",
+    )
+    @pytest.mark.parametrize(
+        "protocol",
+        [
+            "grpc",
+            pytest.param(
+                "http",
+                # NOTE: skip because xfail was causing server to not get cleaned up by test in background
+                marks=pytest.mark.skip(
+                    reason="http not supported decoupled models and model profiling yet"
+                ),
+            ),
+        ],
+    )
+    @pytest.mark.timeout(LLM_TIMEOUT_SECS)
+    def test_llmapi_e2e(self, llmapi_server, protocol):
+        # NOTE: VLLM test models will be passed by the testing infrastructure.
+        # Only a single model will be passed per test to enable tests to run concurrently.
+        model = os.environ.get("LLMAPI_MODEL")
+        assert model is not None, "LLMAPI_MODEL env var must be set!"
+        # Source is optional if using a "known: model"
+        source = os.environ.get("MODEL_SOURCE")
+        TritonCommands._clear()
+        TritonCommands._import(model, source=source, backend="llmapi")
+        llmapi_server.start()
+        TritonCommands._infer(model, prompt=PROMPT, protocol=protocol)
+        # llmapi "backend" is not using the exact same api format as tensorrtllm(tensorrt_llm_bls) backend or vllm backend,
+        # but genai-perf only supports those two, temporarily set genai-perf's backend as vllm,
+        # since the default request of vllm from genai-perf still work with llmapi "backend"
+        # TODO: update this to llmapi when genai-perf supports the llmapi backend's api format
```

Comment:
```suggestion
        # llmapi "backend" is not using the exact same api format as tensorrtllm(tensorrt_llm_bls) backend or vllm backend,
        # but genai-perf only supports those two. So we temporarily set genai-perf's backend as vllm,
        # since the default request of vllm from genai-perf still work with llmapi "backend".
        # TODO: update this to llmapi when genai-perf supports the llmapi backend's api format
```

Suggest we move this comment at the beginning to draw more attention.

Reply:
Thanks, updated.